raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -259,6 +263,13 @@ class IntColorMap(breaksToColors: Map[Int, Int], val options: Options = Options.
     h.foreachValue(z => ch.setItem(z, map(z)))
     new IntCachedColorMap(orderedColors, ch, options)
   }
+
+  lazy val breaksString: String = {
```

Comment:
I wasn't thinking vals...why not defs? Is there ever a reason to hold them in memory after calculation (e.g. its an expensive operation that will be reused many times to justify the instance footprint)?